.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
We've tried some which has these things checked in and some which have not and overall found that checking in works pretty well and also helps ensure everyone is coding in a consistent way (e.g. number of spaces for each tab, if 2 was adopted instead. 2 seems to be pretty common/standard for web projects in particular).